density_scaler/src/server.rs

```diff
@@ -47,19 +46,6 @@ impl Server {
             }
 
             tokio::select! {
-                query = queries.recv() => match query {
-                    Some(QueryMsg{hex, response: tx}) => {
-                        let resp = self
-                                   .scaling_map
-                                   .get(&hex)
-                                   .map(|scale| scale.to_owned());
-                        tx.send(resp)
-                    },
-                    None => {
-                        tracing::warn!("query channel closed");
-                        return Ok(())
-                    }
-                },
                 // _ = trigger_timer.tick() => self.refresh_scaling_map().await?,
```

Comment:
should this be uncommented now?

Reply:
This is from the base branch, Jeff is testing something this weekend